File: api/webapp.py
def handler(request):
    if request.method == "OPTIONS":
        return "", 204, {
            "Access-Control-Allow-Origin": "*",
            "Access-Control-Allow-Headers": "Content-Type",
            "Access-Control-Allow-Methods": "POST, OPTIONS",
        }
    data = request.get_json() or {}
    print("=== WEBAPP REQUEST ===", data)
    return {"ok": True, "received": data}, 200, {
        "Access-Control-Allow-Origin": "*"
    }

File: api/webapp.js
export default function handler(req, res) {
  // CORS preflight
  if (req.method === "OPTIONS") {
    res.setHeader("Access-Control-Allow-Origin", "*");
    res.setHeader("Access-Control-Allow-Headers", "Content-Type");
    res.setHeader("Access-Control-Allow-Methods", "POST, OPTIONS");
    return res.status(204).end();
  }

  console.log("=== WEBAPP REQUEST ===", req.body);
  res.setHeader("Access-Control-Allow-Origin", "*");
  res.status(200).json({ ok: true, received: req.body || null });
}
